app/javascript/components/common/LayoutHeader.js

```diff
@@ -32,6 +32,13 @@ class LayoutHeader extends PureComponent {
     componentDidMount() {
         this.updateMenuVisibility();
         this.targetElement = document.querySelector('.header__mobile-menu ul');
+        //getSocial 
+        const po = document.createElement("script"); 
```

Comment:
Is this the best way to do it? I.e. couldn't you just add a `script` tag?

Reply:
not with react.
as far as I understand it we need to append it after the components is mounted. but let's try to get a second opinion.

Reply:
I'm fine with that explanation.